docs/references/creating-a-release.md

```diff
@@ -10,41 +10,86 @@ We must do due diligence in testing our packages before any new major/minor or b
 
-This process has the following benefits:
+1. Options for this workflow:
+    1. Branch - This is the branch that the release will be created from. Default option is from `main`.
+    2. Bump Type - This is the type of release that will be created, the options for this are:
+        - `beta`
+        - `stable-minor`
+        - `stable-patch`
 
-- The release branch never diverges off of `main`. In theory, it is possible to abandon the release branch at any point and create a new one off of `main` without losing work.
--  All PR reviews happen on `main`, and the cherry-pick PR simply requires a sign-off. This avoids non-trivial merge conflicts when the release branch is eventually merged back into `main`.
+This workflow will:
+
+1. Create a `prerelease/<release-tag>` branch, bump the package version for `@azure/communication-react` as appropriate and collect all change files into a changelog.
+    1. This step will also create a file called `GROOMME.md` which should be deleted when completing the PR back into the pre-release branch to signify that the changelog has been manually groomed.
+1. Create a Pull Request into the `prerelease/<release-tag>` for manually summarizing the changes collected in the changelog. *Merge this PR before going to step 2*.
+
+For example, when creating a release off of `main` tagged `3.3.0`, the following branches and Pull Requests will be created:
+
+```mermaid
+graph LR
+  main[branch: main]
+  prerelease[branch: prerelease/3.3.0<br/>- bump version to 3.3.0<br/>- collect changelog<br/>-trigger string translations]
+  pr[branch:groom_changelog_3.0.0]
+
+  main -->|Create Branch| prerelease
+  prerelease -->|Create Branch| pr
+  pr -.-o|Create Pull Request| prerelease
+```
+
+### Step 2: Create release branch
+
+Use the [create-release-branch](https://github.com/Azure/communication-ui-library/actions/workflows/create-release-branch.yml) github action to trigger the release branch creation workflow.
+
+1. Options for GH Action:
+    1. Pre-release branch - This is the pre-release branch that was created in the previous workflow. This action should be only done after the changelog is groomed and merged back into the pre-release branch to avoid cherry picking.
+
+This workflow will:
+
+1. Create a Pull Request to merge the prerelease branch back into the base branch.
+  1. For stable release only: Before creating the Pull Request, @azure/communiation-react will be version bumped once again to a `-beta.0` version (this ensures that we can continue to add changes of `prerelease`) type.
+1. Create a new release branch off of the prerelease branch. This branch will be used for the eventual release, but *it will never be merged back in the base branch*.
+  1. On the release branch will check what kind of release this is, beta, stable-minor, or stable-patch.
```

Comment:
This list is a bit verbose, and like to bitrot as compared to the workflow file. Can you summarize this a bit?